recipes/onnx/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+export ONNX_BINARY_BUILD=1
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.9
```

Comment:
`toolchain` should already do this.

That makes it probably cleaner to drop `build.sh` entirely and just do:

```yaml
build:
  script: ONNX_BINARY_BUILD=1 python setup.py install --single-version-externally-managed --record=record.txt
```